geode-experimental-driver/src/integrationTest/java/org/apache/geode/experimental/driver/SSLTest.java

```diff
@@ -179,7 +179,7 @@ public void driverCanConnectWithOneWayAuthentication() throws Exception {
 
   @Test
   public void driverCannotConnectIfProtocolsMismatch() throws Exception {
-    startLocator(SERVER_KEY_STORE, SERVER_TRUST_STORE, true, "TLSv1.1", "any");
+    startLocator(SERVER_KEY_STORE, SERVER_TRUST_STORE, true, "TLSv1.2", "any");
```

Comment:
since the test is trying to create a mismatch (by choosing TLS1.1 and TLS1.2), it isn't the same test anymore to use 1.2 vs 1.2.  Maybe it should now be changed to TLSv1.3 on line 187?

Reply:
good point...